### ISSUE TITLE
METEOR@1.6.1-beta.18 does not handle babel 'exportDefaultFrom'

### ISSUE THREAD

**Issue by menelike** [meteor/meteor#9471]
When using `export from` syntax (https://babeljs.io/docs/plugins/transform-export-extensions/) like

**index.js**
```
export Foo from './Bar'
```
the build crashes and throws `This experimental syntax requires enabling the parser plugin: 'exportDefaultFrom' (1:7)`.

I can confirm that it works on `METEOR@1.6.1-beta.5`.

**Comment by StorytellerCZ** [comment]
Doesn't seem like the given plugin has been upgraded to Babel 7 yet.

**Comment by benjamn** [comment]
This should be easy to fix, though in the meantime you could also use
```js
export { default as Foo } from "./Bar"
```

**Comment by benjamn** [comment]
This should be fixed if you run `meteor update --release 1.6.1-beta.19`. Thanks for catching this!
